src/ShaderGlossy.h

```diff
@@ -30,16 +30,57 @@ class CShaderGlossy : public CShaderPhong {
 		Vec3f normal = ray.hit->getNormal(ray);									// shading normal
 
 		// --- PUT YOUR CODE HERE ---
```

Comment:
I suspect plagiarism here.

Reply:
I don't plagiarize. This problem was discussed in the tutorial held by the TA, and I did it as it was there.